gallery/src/components/NxLoadError/NxLoadErrorPage.tsx

```diff
@@ -56,7 +56,17 @@ const NxLoadErrorPage = () =>
             <NxTable.Cell>Function</NxTable.Cell>
             <NxTable.Cell>No</NxTable.Cell>
             <NxTable.Cell>
-              If this is defined, a Retry button will be rendered which executes this function when clicked
+              If this is defined, a Retry button will be rendered with type "button" or "submit" if
```

Comment:
I think the phrasing here would be cleaner if the mention of the `type` was removed.  That can be fully explained in the `useSubmitRetry` description instead.  Though you might also want to mention here that this prop should typically not be used if `useSubmitRetry` is true, since you'll typically use the form's `onSubmit` instead.